File: Python/Tasks/task.py
class Task: 
    def __init__ (self, name):
        self.name = name
        self.completed = False

    def change_name(self, new_name):
        self.name = new_name
    
    def complete(self):
        self.completed = True

    def __str__(self):
        status = "✓" if self.completed else "✗"
        return f"{self.name} [{status}]"

File: Kotlin/Tasks/src/tasks/Task.kt
package tasks

class Task (private var name: String) {
    private var completed = false

    fun getName (): String {
        return name
    }

    // returns either a checkmark or an x if the task is completed or not
    fun getCompleted (): String {
        if (completed) {
            return "✓"
        }
        return "✗"
    }

    fun changeName (newName: String) {
        name = newName
    }

    fun toggleComplete () {
        completed = !completed
    }
}